src/main/java/com/whitesource/remediation/encoder/Encode.java

```diff
@@ -0,0 +1,209 @@
+package com.whitesource.remediation.encoder;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.owasp.esapi.codecs.UnixCodec;
+import org.owasp.esapi.codecs.WindowsCodec;
+
+/**
+ * Remediation Solver static class written by WhiteSource with the community ❤. Here you can find
+ * wrapper functions to secure unsafe operations in your code.
+ */
+public class Encode {
+
+  /**
+   * Encoding operating system parameters.
+   *
+   * @param params The parameters for the operating systems.
+   * @return Encoded parameters.
+   */
+  public static String OSParameterEncoder(@NonNull final String params)
```

Comment:
Should be param not params, maybe add an overloading funtion that accepts a collection of strings